lib/MusicBrainz/Server/Edit/Historic/Relationship.pm

```diff
@@ -220,18 +220,18 @@ sub _display_relationships {
             my $entity1_id = $_->{entity1_id};
             my $entity0 = $loaded->{ $model0 }{ $entity0_id } ||
                 $self->c->model($model0)->_entity_class->new(
-                    id => $entity0_id,
+                    id => $entity0_id // 0,
```

Comment:
Even though it fix the ISE, it still causes the warning “Use of uninitialized value `$entity0_id` in hash element”.

I would suggest to make your change once and for all when setting the `*_id` variables instead:

```
            my $entity0_type = $_->{entity0_type} // 0;
            my $entity1_type = $_->{entity1_type} // 0;
```